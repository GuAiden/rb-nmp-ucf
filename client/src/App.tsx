import React from 'react';
import './App.css';
import Navbar from './components/Navbar';
import MenuSwitcher from './components/MenuSwitcher';
import InputForm from './components/Forms/InputForm.tsx';

const App: React.FC = () => (
  <div className="App">
    <Navbar />
    <MenuSwitcher />
    <InputForm />
    <header className="App-header">
      <p>
        Hey <code>peeps</code>
      </p>
    </header>
  </div>
);

export default App;
