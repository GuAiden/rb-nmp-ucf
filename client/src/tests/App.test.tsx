import React from 'react';
import { render, screen } from '@testing-library/react';
import App from '../App';

test('renders navbar', () => {
  render(<App />);
  const linkElement = screen.getByAltText('redbacklogo');
  expect(linkElement).toBeInTheDocument();
});
