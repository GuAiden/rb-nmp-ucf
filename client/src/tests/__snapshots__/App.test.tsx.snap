// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders forms component correctly without props 1`] = `ShallowWrapper {}`;
